src/Context/index.js

```diff
@@ -2,6 +2,7 @@ import { AuthProvider, useAuth } from "./authContext";
 import { ActivitiesProvider, useActivities } from "./activitiesContext";
 import { ToastProvider, useToast } from "./toastContext";
 import { useVideo, VideoProvider } from "./videosContext";
+import { PlayListProvider, usePlaylist } from "./playlistContext";
 
```

Comment:
we can write export * from "/..." instead of importing and exporting

Reply:
are nice
thanks for telling